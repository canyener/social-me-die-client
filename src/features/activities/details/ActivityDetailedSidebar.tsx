import React, { Fragment } from 'react'
import { Link } from 'react-router-dom'
import { Segment, List, Item, Label, Image } from 'semantic-ui-react'

const ActivityDetailedSidebar = () => {
  return (
      <Fragment>
        <Segment
          textAlign='center'
          style={{ border: 'none' }}
          attached='top'
          secondary
          inverted
          color='teal'
        >
          3 People Going
        </Segment>
        <Segment attached>
          <List relaxed divided>
            <Item style={{ position: 'relative' }}>
              <Label
                style={{ position: 'absolute' }}
                color='orange'
                ribbon='right'
              >
                Host
              </Label>
              <Image size='tiny' src={'/assets/user.png'} />
              <Item.Content verticalAlign='middle'>
                <Item.Header as='h3'>
                  <Link to={`#`}>Can</Link>
                </Item.Header>
                <Item.Extra style={{ color: 'orange' }}>Following</Item.Extra>
              </Item.Content>
            </Item>
  
            <Item style={{ position: 'relative' }}>
              <Image size='tiny' src={'/assets/user.png'} />
              <Item.Content verticalAlign='middle'>
                <Item.Header as='h3'>
                  <Link to={`#`}>Cem</Link>
                </Item.Header>
                <Item.Extra style={{ color: 'orange' }}>Following</Item.Extra>
              </Item.Content>
            </Item>
  
            <Item style={{ position: 'relative' }}>
              <Image size='tiny' src={'/assets/user.png'} />
              <Item.Content verticalAlign='middle'>
                <Item.Header as='h3'>
                  <Link to={`#`}>Onur</Link>
                </Item.Header>
              </Item.Content>
            </Item>
          </List>
        </Segment>
      </Fragment>
  )
}

export default ActivityDetailedSidebar