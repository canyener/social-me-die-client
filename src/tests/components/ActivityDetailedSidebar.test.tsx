import { shallow } from 'enzyme'

import ActivityDetailedSidebar from '../../features/activities/details/ActivityDetailedSidebar'

 test('Should render ActivityDetailedSidebar correctly', () => {
   const wrapper = shallow(<ActivityDetailedSidebar />)

   expect(wrapper).toMatchSnapshot()
 })