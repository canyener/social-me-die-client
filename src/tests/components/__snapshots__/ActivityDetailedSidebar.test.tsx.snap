// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ActivityDetailedSidebar correctly 1`] = `
<Fragment>
  <Segment
    attached="top"
    color="teal"
    inverted={true}
    secondary={true}
    style={
      Object {
        "border": "none",
      }
    }
    textAlign="center"
  >
    3 People Going
  </Segment>
  <Segment
    attached={true}
  >
    <List
      divided={true}
      relaxed={true}
    >
      <Item
        style={
          Object {
            "position": "relative",
          }
        }
      >
        <Label
          color="orange"
          ribbon="right"
          style={
            Object {
              "position": "absolute",
            }
          }
        >
          Host
        </Label>
        <Image
          as="img"
          size="tiny"
          src="/assets/user.png"
          ui={true}
        />
        <ItemContent
          verticalAlign="middle"
        >
          <ItemHeader
            as="h3"
          >
            <Link
              to="#"
            >
              Can
            </Link>
          </ItemHeader>
          <ItemExtra
            style={
              Object {
                "color": "orange",
              }
            }
          >
            Following
          </ItemExtra>
        </ItemContent>
      </Item>
      <Item
        style={
          Object {
            "position": "relative",
          }
        }
      >
        <Image
          as="img"
          size="tiny"
          src="/assets/user.png"
          ui={true}
        />
        <ItemContent
          verticalAlign="middle"
        >
          <ItemHeader
            as="h3"
          >
            <Link
              to="#"
            >
              Cem
            </Link>
          </ItemHeader>
          <ItemExtra
            style={
              Object {
                "color": "orange",
              }
            }
          >
            Following
          </ItemExtra>
        </ItemContent>
      </Item>
      <Item
        style={
          Object {
            "position": "relative",
          }
        }
      >
        <Image
          as="img"
          size="tiny"
          src="/assets/user.png"
          ui={true}
        />
        <ItemContent
          verticalAlign="middle"
        >
          <ItemHeader
            as="h3"
          >
            <Link
              to="#"
            >
              Onur
            </Link>
          </ItemHeader>
        </ItemContent>
      </Item>
    </List>
  </Segment>
</Fragment>
`;
